apps/dapp/src/types/typechain/factories/RAMOSGoerli__factory.ts

```diff
@@ -0,0 +1,1061 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
Yeah unfortunately the SOP for the temple dapp has been copypasta-ing the typechain generated from the protocol repo

I think (hopefully) this will change in the next iteration of the dapp.

Done, now using mainnet contracts :+1: